sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/__init__.py

```diff
@@ -11,39 +11,40 @@
 from ._data_lake_service_client import DataLakeServiceClient
 from ._data_lake_lease import DataLakeLeaseClient
 from ._models import (
-    LocationMode,
```

Comment:
Thanks for sorting these! :)

Reply:
Of course! I also used my thinking cap yesterday and used CTRL+SHIFT+P --> `Sort Lines Ascending` in VS Code, rather than manually alphabetizing as I have been 😆

Reply:
Ohhhh, I didn't know you could do that! I always do it manually 😄